From 822e1f53de3b8d6a82c4f709f757ba1060891e40 Mon Sep 17 00:00:00 2001
From: sai surya teja <sssteja@habana.ai>
Date: Thu, 27 Oct 2022 17:08:50 +0300
Subject: [PATCH] [SW-109189] gaudi3: fix hbw rr config for pcie_close

There was a typo in security_rr_close_pcie_config() where rr_ctx_lbw
is used for security_rr_config_pcie_hbw. Due to this HBW doesn't get
blocked in bootrom normal flow.
This patch fixes the above mentioned issue by replacing rr_ctx_lbw with
rr_ctx_hbw.

Change-Id: Iac194906e1698e67ae7c2cbca7ec5fb1041ce460
---

diff --git a/arch/gaudi3/arc/security.c b/arch/gaudi3/arc/security.c
index 196ecbd..cfa0f53 100644
--- a/arch/gaudi3/arc/security.c
+++ b/arch/gaudi3/arc/security.c
@@ -732,7 +732,7 @@
 
 	security_rr_config_pcie_lbw(&rr_ctx_lbw[RR_CLOSE_ALL]);
 
-	security_rr_config_pcie_hbw(&rr_ctx_lbw[RR_CLOSE_ALL]);
+	security_rr_config_pcie_hbw(&rr_ctx_hbw[RR_CLOSE_ALL]);
 }
 
 void arcboot_rom_security_init(void)
